src/components/TokenCard.tsx

```diff
@@ -32,7 +32,7 @@ const TokenCard: FC<TokenCardProps> = ({
     }
   }, []);
 
-  const imageUrl = useCheckImageExists(getImageURL(imagePath));
+  const imageUrl = useCheckImageExists(image.fullUrl || image.url);
```

Comment:
Are you sure that old urls will come in pure format, not in json? getImageURL handles the 'json' case

Reply:
Yep, It's correct.
You can check this https://scan-api.demo.uniquenetwork.dev/v1/graphql


```GQL
query getTokens {
  tokens(where: { collection_id: { _eq: 1987 }, token_id: { _eq: 17 } }) {
    data {
      owner
      token_id
      token_name
      image
    }
  }
}
```